src/test/resources/org/kohsuke/github/GHBranchProtectionTest/wiremock/testGetProtection/mappings/repos_github-api-test-org_temp-testgetprotection_branches_master_protection-4.json

```diff
@@ -0,0 +1,40 @@
+{
+  "id": "1d395564-9154-440b-ade4-b18cb8b970cc",
+  "name": "repos_github-api-test-org_temp-testgetprotection_branches_master_protection",
+  "request": {
+    "url": "/repos/github-api-test-org/temp-testGetProtection/branches/master/protection",
+    "method": "GET",
+    "headers": {
+      "Accept": {
+        "equalTo": "text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2"
+      }
+    }
+  },
+  "response": {
+    "status": 404,
+    "body": "{\"message\":\"Branch not protected\",\"documentation_url\":\"https://developer.github.com/v3/repos/branches/#get-branch-protection\"}",
```

Comment:
This is where the error is recorded. Changing the `protected` where you didn't doesn't change this result being returned.